packages/components/themes/base.js

```diff
@@ -698,7 +698,8 @@ const Base = {
       background: white,
 
       border: "1px solid",
-      borderRadius: "0 3px 3px 0",
+      borderRadiusLtr: "0 3px 3px 0",
```

Comment:
It may be worth using the getCorrectFourValuesStyle utility

Reply:
Change to getCorrectBorderRadius utility